x/evm/statedb/statedb.go

```diff
@@ -396,6 +396,16 @@ func (s *StateDB) AddAddressToAccessList(addr common.Address) {
 	}
 }
 
+// GetAddressToAccessList return full access list
+func (s *StateDB) GetAddressToAccessList() *accessList {
```

Comment:
Currently not sure on the implication of the `unexported-return` warning - this seems like bad practice. Is there maybe a better way to handle this?
```suggestion
// GetAccessList return full access list
func (s *StateDB) GetAccessList() *accessList {
```